simapp/test_helpers.go

```diff
@@ -104,6 +144,35 @@ func AddTestAddrs(app *SimApp, ctx sdk.Context, accNum int, accAmt sdk.Int) []sd
 	return testAddrs
 }
 
+func ConvertAddrsToValAddrs(addrs []sdk.AccAddress) (valAddrs []sdk.ValAddress) {
+	for _, addr := range addrs {
+		valAddrs = append(valAddrs, sdk.ValAddress(addr))
+	}
+
+	return
+}
+
+func TestAddr(addr string, bech string) sdk.AccAddress {
```

Comment:
We should not panic here. Is this for testing purposes? If so, provide a `*testing.T` and fail using that.